src/components/DocumentationTopic/DocumentationNav/LanguageToggle.vue

```diff
@@ -315,7 +316,7 @@ $nav-menu-toggle-label-margin: 6px;
     display: inline-block;
 
     &:not(:first-child) {
-      border-left: 1px solid dark-color(fill-gray-tertiary);
+      border-left: 1px solid var(--color-fill-gray-tertiary);
```

Comment:
I think we want the `dark-color` variant here? The current color contrast of the divider line looks too weak to me.
<img width="613" alt="image" src="https://github.com/apple/swift-docc-render/assets/87735557/610a8360-3500-4cc3-af2b-36a360fd3b55">

Reply:
Edit: I meant that we should pick a different color. `fill-gray-tertiary` is darker in dark mode than in light mode. I would expect the opposite.

Reply:
You are right! Fixed: https://github.com/apple/swift-docc-render/pull/781/commits/8fa580cb913eccfd534461129c32d39fb56432d2

Reply:
![Screenshot 2024-03-14 at 22 55 18](https://github.com/apple/swift-docc-render/assets/8567677/8178d213-2824-42b1-bf3d-c19588e409e9)